test/e2e/expected_output/001_annotations.txt

```diff
@@ -0,0 +1,124 @@
+apps {
```

Comment:
New e2e tests work by creating `actual_output/XXX.txt` from `input/XXX.syl` and comparing it to `expected_output/XXX.txt`

Reply:
Should go into /tmp, but that can be a low-pri issue for a future PR.

Reply:
I assume you mean the `actual_output` directory should go into `/tmp`?
That wouldn't work on Windows, so I would have to request a Python tmp-dir which is random / different each time.
I've added `tests/e2e/actual_output` to `.gitignore` so it doesn't pollute the repo. The reason why I like `expected_output` and `actual_output` side by sider is that I can use a dir diff tool more easily this way I find.

What's the reason for you suggesting it should go into `/tmp`?

Reply:
I prefer to keep the source tree empty of uncommitted output. It's a pretty minor thing, so let's park it for now and discuss later.

Reply:
ok